File: dev-superior-algoritmo-e-logica-de-programacao/cpp/exercicios/pagamento.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main() {
    string nome;
    double valorHora, pagamento;
    int horasTrabalhadas;

    cout << "Nome: ";
    getline(cin, nome);

    cout << "Valor por hora: ";
    cin >> valorHora;

    cout << "Horas trabalhadas: ";
    cin >> horasTrabalhadas;

    pagamento = valorHora * horasTrabalhadas;

    cout << fixed << setprecision(2);
    cout << "O pagamento para " << nome << " deve ser " << pagamento << endl;

	return 0;
}

File: nelio-alves/algoritmo-e-logica-de-programacao/python/exercicios/pagamento.py
nome: str
valorHora: float; pagamento: float
horasTrabalhadas: int

nome = str(input("Nome: "))
valorHora = float(input("Valor por hora: "))
horasTrabalhadas = int(input("Horas trabalhadas: "))

pagamento = valorHora * horasTrabalhadas

print(f"O pagamento para {nome} deve ser {pagamento:.2f}")
